validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/synccommittee/SyncCommitteeAggregationDutyTest.java

```diff
@@ -68,7 +66,7 @@
   private final BLSSignature nonAggregatorSignature =
       BLSSignature.fromBytesCompressed(
           Bytes.fromHexString(
-              "0xae401f767ab1917f925fe299ad51a57d52f7cc80deb1cc20fa2b3aa983e4e4d23056d79f01f3c97e29c8905da17e70e30c2a3f6bdd83dbc4ddf530e02e8f4d7ba22260e12e5f5fe7875b48e79660615b275597e87b2d33e076664b3da1737852"));
+              "0xce401f767ab1917f925fe299ad51a57d52f7cc80deb1cc20fa2b3aa983e4e4d23056d79f01f3c97e29c8905da17e70e30c2a3f6bdd83dbc4ddf530e02e8f4d7ba22260e12e5f5fe7875b48e79660615b275597e87b2d33e076664b3da1737852"));
```

Comment:
The modulus math changed when I removed the customized value for `TARGET_AGGREGATORS_PER_SYNC_SUBCOMMITTEE` - just tweaked this variable until I found a value that would fail the `SyncCommitteeUtil.isSyncCommitteeAggregator()` check.